tests/AutoReview/ProjectCodeTest.php

```diff
@@ -339,6 +339,7 @@ private function getTestClasses()
             ->in(__DIR__.'/..')
             ->exclude(array(
                 'Fixtures',
+                'Test',
```

Comment:
I believe they were in `src` for that reason. If you still want to support them for external usage, then moving them outside `src` does not make sense to me.